Easydict.xcodeproj/project.pbxproj

```diff
@@ -3469,7 +3469,7 @@
 				COPY_PHASE_STRIP = NO;
 				CURRENT_PROJECT_VERSION = 33;
 				DEAD_CODE_STRIPPING = YES;
-				DEVELOPMENT_TEAM = "$(DEVELOPMENT_TEAM)";
+				DEVELOPMENT_TEAM = XXM539CLCH;
```

Comment:
Why change this?

Reply:
<img width="1093" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/3147eb1c-b990-417b-bc83-2bc9cf4269a0">

Reply:
submit by mistake, let me revert this🤡